firmware/quadruna/VC/src/app/states/app_inverterOnState.c

```diff
@@ -45,37 +50,43 @@ static void inverterOnStateRunOnTick100Hz(void)
     const bool inverter_has_fault  = app_faultCheck_checkInverters();
     const bool all_states_ok       = !(any_board_has_fault || inverter_has_fault);
 
-    const bool curr_start_switch_on     = app_canRx_CRIT_StartSwitch_get();
-    const bool was_start_switch_enabled = !prev_start_switch_pos && curr_start_switch_on;
-    const bool is_brake_actuated        = app_canRx_FSM_BrakeActuated_get();
+    const bool bms_in_drive = app_canRx_BMS_State_get() == BMS_DRIVE_STATE;
 
-    const bool bms_ready_for_drive = app_canRx_BMS_State_get() == BMS_DRIVE_STATE;
-    const bool hv_support_lost =
-        app_canRx_BMS_State_get() == BMS_INIT_STATE || app_canRx_BMS_State_get() == BMS_FAULT_STATE;
+    const bool is_invr_on = app_canRx_INVL_VsmState_get() == INVERTER_VSM_READY_STATE;
+    const bool is_invl_on = app_canRx_INVL_VsmState_get() == INVERTER_VSM_READY_STATE;
+    const bool is_inv_on  = is_invr_on && is_invl_on;
 
-    if (hv_support_lost)
+    if (!power_manager_inverter_on.efuses[EFUSE_CHANNEL_INV_R])
+    {
+        io_efuse_setChannel(EFUSE_CHANNEL_INV_R, true);
+        power_manager_inverter_on.efuses[EFUSE_CHANNEL_INV_R] = true;
+        app_canTx_VC_RightInverterEnable_set(true);
```

Comment:
This looks really good. One note is that we dont want to enable inverters yet in this state